src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/HttpApi/ApiClientExtensions.cs

```diff
@@ -516,16 +516,24 @@ public static async Task<OperationStatusResponse> PollOperationToCompletion(this
         /// </summary>
         public static Task<OperationResponse> CaptureParametersAsync(this ApiClient client, int pid, TimeSpan duration, CaptureParametersConfiguration config)
         {
-            return client.CaptureParametersAsync(pid, duration, config, TestTimeouts.HttpApi);
+            return client.CaptureParametersAsync(pid, duration, string.Empty, config, TestTimeouts.HttpApi);
         }
 
         /// <summary>
         /// POST /parameters
         /// </summary>
-        public static async Task<OperationResponse> CaptureParametersAsync(this ApiClient client, int pid, TimeSpan duration, CaptureParametersConfiguration config, TimeSpan timeout)
+        public static Task<OperationResponse> CaptureParametersAsync(this ApiClient client, int pid, TimeSpan duration, CaptureParametersConfiguration config, string egressProvider)
+        {
+            return client.CaptureParametersAsync(pid, duration, egressProvider, config, TestTimeouts.HttpApi);
+        }
+
+        /// <summary>
+        /// POST /parameters
+        /// </summary>
+        public static async Task<OperationResponse> CaptureParametersAsync(this ApiClient client, int pid, TimeSpan duration, string egressProvider, CaptureParametersConfiguration config, TimeSpan timeout)
```

Comment:
nit: For readability can we keep the ordering of parameters consistent here - it looks like this flips `egressProvider` and `config` compared to the method right above it